packages/engines/nextflow/buildspec.yml

```
@@ -5,7 +5,7 @@ env:
   variables:
     # These variables may be over-ridden as appropriate by the CI/CD pipeline
     NEXTFLOW_IMAGE_NAME: "nextflow"
-    NEXTFLOW_VERSION: "21.04.3"
+    NEXTFLOW_VERSION: "22.04.1"
```

https://github.com/nextflow-io/nextflow/releases/tag/v22.04.3
```suggestion
    NEXTFLOW_VERSION: "22.04.3"
```